### DISCUSSION TITLE
Add "node_labels" to users without multiples roles ?

### DISCUSSION THREAD

**Discussion by fili-po** [gravitational/teleport#17294] [Q&A] [answered]
Hi,

Can I add user node_labels in user YAML creation file ?
Same feature than "internal.logins" but with "internal.node_labels" but it's not recognized

My problem :
User "user1" acces node_labels "foobar1" only
User "user2" acces node_labels "foobar2" only
User "user3" acces node_labels "foobar3" only
Same SSH login in all nodes

But I only want to create one role because I have a large number of users

Thanks
Filipo

**Comment by webvictim** [comment, marked as answer]
This isn't currently supported. There is a feature request open to add an `{{internal.username}}` trait to use in roles: https://github.com/gravitational/teleport/issues/17616
